### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by nemchik**
Logo https://github.com/linuxserver/docker-templates/pull/290
cstate https://github.com/linuxserver/cstate/pull/172

### CODE REVIEW

File: jenkins-vars.yml
Comment (Roxedus) [inline jenkins-vars.yml:1]: Could do with a sponsor_links

File: readme-vars.yml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+development_versions_items:
+  - { tag: "latest", desc: "Stable Kimai releases." }
+
+# container parameters
+common_param_env_vars_enabled: true
+param_container_name: "{{ project_name }}"
+param_usage_include_env: true
+param_env_vars:
+  - { env_var: "DATABASE_URL", env_value: "mysql://your_db_user:your_db_pass@your_db_host:3306/your_db_name", desc: "Configure your database connection"}
+  - { env_var: "TZ", env_value: "Europe/London", desc: "Specify a timezone to use EG Europe/London.", name: "Timezone" }
```
Comment (Roxedus) [inline readme-vars.yml]: redundant

File: jenkins-vars.yml
Comment (nemchik) [reply jenkins-vars.yml:1]: Resolved in https://github.com/linuxserver/docker-kimai/pull/1/commits/7dbc3285a4bc2fa01fa9686ff434e8343e20fe10

File: readme-vars.yml (hunk 1 above)
Comment (nemchik) [reply readme-vars.yml]: Resolved in https://github.com/linuxserver/docker-kimai/pull/1/commits/f6288f4b26990c5dd9dd5de64949efad68e11986

**Review by Roxedus — APPROVED** [review]

**Pull request merged**